crates/holochain/src/conductor/conductor.rs

```diff
@@ -1025,6 +1025,26 @@ impl Conductor {
             .collect())
     }
 
+    pub(super) async fn find_cell_with_role_alongside_cell(
```

Comment:
this seems like a good way to do it, this is even future-proof for when apps and cells are many-to-many